test/performance/search/search_benchmark.cpp

```diff
@@ -159,7 +159,7 @@ void unidirectional_search_all_collection(benchmark::State & state, options && o
     std::vector<std::vector<seqan3::dna4>> reads;
     for (size_t i = 0; i < set_size; ++i)
     {
-        collection.push_back(seqan3::test::generate_sequence<seqan3::dna4>(o.sequence_length, 0, i));
+        collection.push_back(seqan3::test::generate_sequence<seqan3::dna4>(o.sequence_length / set_size, 0, i));
```

Comment:
This did not work out as planned, but I think it would be very good to adapt the collection tests so that the total compared sequence length is the same between collection and single. It makes comparing the numbers much easier!